.github/workflows/rts.yml

```diff
@@ -88,14 +88,11 @@ jobs:
           git config --global user.name "github-actions[bot]"  
           WORKSPACE=$PWD PROFILE=RTS ./scripts/hudson/narayana.sh
 
+      - name: Zip log files into log.zip
+        run: zip -r log.zip . -i **/*.*log*  **/ObjectStore*/**  **/tx-object-store/** **/*tx-object-store.zip **/target/*surefire-reports*/** **/target/*failsafe-reports*/** -r 
+        
```

Comment:
Github Actions already do this for us. As this workflow doesn't have issues with log files, I don't think we need this extra step